src/tools/dev/scripts/bv_support/bv_qt.sh

```diff
@@ -248,6 +248,11 @@ function apply_qt_patch
                 fi
             fi
         fi
+
+        apply_qt_5101_gcc_9_2_patch
```

Comment:
@brugger1 and @cyrush  I would think this patch would need to be wrapped in a conditional based on OS type and version?

Reply:
good catch, OS type can be linux -- than then gcc + version should be another screen.